app.py

```diff
@@ -48,9 +47,10 @@ class Schedule:
     def __init__(self, github: Github, organizations: List[str] = None, repositories: List[str] = None):
```

Comment:
Just set the default for organizations and repositores to be `[]` here

Reply:
python is known to work unexpectedly here: if you use mutable as a default parameter, it can be modified later, which would affect all the other subsequent calls.

See here e.g.: https://web.archive.org/web/20200221224620/http://effbot.org/zone/default-values.htm

Therefore, for a better practice, I use `None` and then check, instead of mutables as default (even though they are not modified in body).